transactions/mint_nft.cdc

```diff
@@ -1,24 +1,33 @@
 import NonFungibleToken from 0xNFTADDRESS
 import ExampleNFT from 0xNFTCONTRACTADDRESS
 
+// This script uses the NFTMinter resource to mint a new NFT
+// It must be run with the account that has the minter resource
+// stored in /storage/NFTMinter
 
 transaction(recipient: Address) {
     
+    // local variable for storing the minter reference
     let minter: &ExampleNFT.NFTMinter
 
     prepare(signer: AuthAccount) {
 
-        self.minter = signer.borrow<&ExampleNFT.NFTMinter>(from: /storage/NFTMinter)!
+        // borrow a reference to the NFTMinter resource in storage
+        self.minter = signer.borrow<&ExampleNFT.NFTMinter>(from: /storage/NFTMinter)?
```

Comment:
Is this correct to have a '?' in addition to '??'?

Reply:
It is still valid, but not necessary, so I'll remove it

Reply:
Interesting – Is this even parsing? `...Minter)?` isn't legal syntax 🤔